MakieCore/src/basic_plots.jl

```diff
@@ -40,6 +40,30 @@ function generic_plot_attributes(attr)
     )
 end
 
+function mixin_generic_plot_attributes()
+    @DocumentedAttributes begin
+        transformation = automatic
```

Comment:
Should we add `material` here for RPRMakie?  It's not quite a shading attribute, IMO, since it can also control the height through subsurface materials.

Reply:
I remember Simon saying he was not super happy with `material` because it has no way of being supported in the other backends. But I'm not sure if it can all be handled by `color` plus types?

Reply:
It would be a bit difficult to handle colors passed as values while also setting the material properties (metallicity, etc) if they were all packed in `color`, no?

Reply:
could always be `color = RPRColor(values, metallicity = ...)` or whatever. @SimonDanisch what's your opinion?